repository/src/main/java/org/apache/atlas/repository/store/graph/v2/EntityGraphMapper.java

```diff
@@ -196,6 +203,26 @@ public class EntityGraphMapper {
 
     private static final Set<String> excludedTypes = new HashSet<>(Arrays.asList(TYPE_GLOSSARY, TYPE_CATEGORY, TYPE_TERM, TYPE_PRODUCT, TYPE_DOMAIN));
 
+    Configuration configuration;
+
+    {
+        try {
+            configuration = ApplicationProperties.get();
+        } catch (AtlasException e) {
+            throw new RuntimeException(e);
+        }
+    }
+
+    KafkaNotification kfknotif;
+
+    {
+        try {
+            kfknotif = new KafkaNotification(configuration);
```

Comment:
it should be possible to inject KafkaNotification in the constructor
see if that is possible so no need to initialize separate object here